### PR TITLE
docs: add admonition page

### PR DESCRIPTION
**Pull request by bradryanbice**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

### CODE REVIEW

**Review by caseybaggz — COMMENTED** [review]
@bradryanbice we need to add a position to the color.mdx frontMatter so it stays at the top. See other intro pages for the prop to use.

**Comment by bradryanbice** [comment]
> @bradryanbice we need to add a position to the color.mdx frontMatter so it stays at the top. See other intro pages for the prop to use.

That's a miss, Color should be in a different folder. We just have a Design bucket right now, we need separate areas.

File: website/docs/design/admonition.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Used to communicate a state that affects a system, feature, or page.
+
+## Anatomy/Structure
+
+- Contents
+  - Status Icon
+  - Text
+    - Title
+    - Description
+- Close icon
```
Comment (Rykus0) [inline website/docs/design/admonition.md:17]: Close *button*?

File: website/docs/design/admonition.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## Behavior
+
+- Placement
+
+  - Admonitions should be placed at the top of the current content area, to capture the attention of users or be read as quickly as possible.
+  - When associated with forms, Admonitions should be placed at the beginning of the form content.
+
+- Close button
+  - Include the close button when the admonition is communicating a real-time response or status to the user, allowing them to dismiss the message. For in-content admonitions (usually warnings or informative messages) do not include a close button so that these messages aren't lost.
+- Icons
```
Comment (Rykus0) [inline website/docs/design/admonition.md:63]:
```suggestion
- Status Icons
```

File: website/docs/design/admonition.md (hunk 1 above)
Comment (bradryanbice) [reply website/docs/design/admonition.md]: 👍 updated

**Review by caseybaggz — APPROVED** [review]

**Pull request merged**